datalad/support/gitrepo.py

```diff
@@ -1697,7 +1698,10 @@ def _git_custom_command(self, files, cmd_str,
         ----------
         files: list of files
         cmd_str: str or list
-            arbitrary command str. `files` is appended to that string.
+          arbitrary command str. `files` is appended to that string.
+        update_tree: bool
```

Comment:
`update_tree` is imho somewhat misleading suggesting to do something ("update the tree"), if anything let it be `updates_tree`, I will quickly push that change and also add its use in other relevant commands